cmd/thv-registry-api/app/serve.go

```diff
@@ -27,7 +27,11 @@ var serveCmd = &cobra.Command{
 	Use:   "serve",
 	Short: "Start the registry API server",
 	Long: `Start the registry API server to serve MCP registry data.
-The server reads registry data from ConfigMaps and provides REST endpoints for clients.`,
+The server can read registry data from either:
```

Comment:
just a minor thing. In our proxyrunner we have a flag also to read things from configmap, and we call it "--from-configmap". Not important, because we are going to remove this flag, but just raising for naming consistency